services/static.js

```diff
@@ -271,8 +271,14 @@ function encodePosition(position) {
   if (position === 'auto') return 'auto';
 
   return position.coordinates
-    .concat([position.zoom, position.bearing, position.pitch])
-    .filter(Boolean)
+    .concat([
+      position.zoom,
+      position.pitch && !position.bearing ? 0 : position.bearing, // if pitch is set, bearing must be 0
```

Comment:
Just small suggestion for the benefit of our future selves!

```suggestion
      position.pitch && !position.bearing ? 0 : position.bearing, // if pitch is set, but bearing is not, bearing must be 0
```